prisma/seeds/prisma-seed.graphql

```diff
@@ -450,405 +411,6 @@ mutation {
               ]
             }
           }
-          {
```

Comment:
we should not be removing all the other seeded products. they are needed for styling that product list view to make sure everything looks ok when there are many products.

Reply:
The seed changes are required to be compatible with the schema changes the stakeholder requested, and are also the foundation for future features such as the user based roles that everything in including code climate integration will ultimately be tied to.